ibm/mas_devops/README.md

```diff
@@ -5,7 +5,7 @@
 
 ## Change Log
 Note that links to pull requests prior to public release of the code (4.0) direct to IBM GitHub Enterprise, and will only be accessible to IBM employees.
-
+- `13.2` Add additional AWS storage support for MVI playbook ([#712](https://github.com/ibm-mas/ansible-devops/pull/712))
```

Comment:
The new line is needed before the list starts, or this won't render correctly .. also the other changelog in docs/changes.md also needs the same update